examples/2node-ixia-ceos.pb.txt

```diff
@@ -0,0 +1,50 @@
+name: "ixia-c"
+nodes: {
+    name: "otg"
+    type: IXIA_TG
+    version: "0.0.1-2770"
+    services: {
+        key: 40051
+        value: {
+            name: "grpc"
```

Comment:
move these to consistent ports as above?

Reply:
could you please clarify this further ? do you mean we need to use some standard TCP port for gRPC and gNMI ?

Reply:
ok, got it; i think you meant making the OTG node definition consistent across all example files. Done !

Reply:
https://github.com/google/kne/pull/138/commits/f322e580c4df8273520c0f98be4dc6397b568ef0